pages/Documents/ConversationalAI/MessagingAgentSDK/overview.md

```diff
@@ -7,6 +7,16 @@ permalink: messaging-agent-sdk-overview.html
 indicator: messaging
 ---
 
+{: .attn-deprecation}
```

Comment:
Please delete {: .attn-deprecation} on line 10. It's unnecessary and doesn't work

Reply:
Done